examples/nightlight/nightlight.py

```diff
@@ -0,0 +1,98 @@
+#!/bin/env python
```

Comment:
Throws a `bad interpreter` error on my Raspbian Lite install. Should be `/usr/bin/env python`.